src/shell/lisa_shell

```diff
@@ -93,10 +93,49 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git update-index -q --refresh
+ret=$?
+if [ $ret -ne 0 ]; then
+    echo "lisa internal error: git update-index failed.  Please report it: https://github.com/ARM-software/lisa/issues"
+    return $ret
+fi
+
+if ! git diff-index --quiet HEAD; then
+    echo "There are outstanding uncommitted changes."
+    echo "Please, commit your changes or stash them before you can update lisa"
+    return 1
+fi
+
+curr_commit=$(git rev-parse HEAD)
+if ! git merge-base --is-ancestor $curr_commit origin/master; then
+    echo "You have committed changes that are not part of origin/master"
+    echo "Please move to the master branch"
```

Comment:
"Please move to the master branch..." I would add also  "before running the lisa-update command"